library/std/src/net/ip.rs

```diff
@@ -485,22 +472,22 @@ impl Ipv4Addr {
     ///
     /// The following return false:
     ///
-    /// - private addresses (see [`is_private()`](#method.is_private))
-    /// - the loopback address (see [`is_loopback()`](#method.is_loopback))
-    /// - the link-local address (see [`is_link_local()`](#method.is_link_local))
-    /// - the broadcast address (see [`is_broadcast()`](#method.is_broadcast))
-    /// - addresses used for documentation (see [`is_documentation()`](#method.is_documentation))
-    /// - the unspecified address (see [`is_unspecified()`](#method.is_unspecified)), and the whole
+    /// - private addresses (see [`Ipv4Addr::is_private`])
+    /// - the loopback address (see [`Ipv4Addr::is_loopback`])
+    /// - the link-local address (see [`Ipv4Addr::is_link_local`])
+    /// - the broadcast address (see [`Ipv4Addr::is_broadcast`])
+    /// - addresses used for documentation (see [`Ipv4Addr::is_documentation`])
+    /// - the unspecified address (see [`Ipv4Addr::is_unspecified`]), and the whole
```

Comment:
I think it would be nice to keep the `()` so it's clear it's a method. As a bonus it makes the doc pass insignificantly faster :P
```suggestion
    /// - private addresses (see [`Ipv4Addr::is_private()`])
    /// - the loopback address (see [`Ipv4Addr::is_loopback()`])
    /// - the link-local address (see [`Ipv4Addr::is_link_local()`])
    /// - the broadcast address (see [`Ipv4Addr::is_broadcast()`])
    /// - addresses used for documentation (see [`Ipv4Addr::is_documentation()`])
    /// - the unspecified address (see [`Ipv4Addr::is_unspecified()`]), and the whole
```

Reply:
Should I do it for every link to a method in all the files ?

Reply:
Only when it makes the docs more clear. The performance benefit really is insignificant, don't worry too much about that.